cloudflare/resource_cloudflare_access_group.go

```diff
@@ -0,0 +1,255 @@
+package cloudflare
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	cloudflare "github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceCloudflareAccessGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCloudflareAccessGroupCreate,
+		Read:   resourceCloudflareAccessGroupRead,
+		Update: resourceCloudflareAccessGroupUpdate,
+		Delete: resourceCloudflareAccessGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceCloudflareAccessGroupImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"account_id": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
If the `account_id` changes, we'll want a new resource created.

```suggestion
				Required: true,
				ForceNew: true,
```